src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyTransportFactory.php

```diff
@@ -41,6 +41,10 @@ public function create(Dsn $dsn): TransportInterface
             $transport->setPort($port);
         }
 
+        if (!empty($token = $dsn->getOption('token'))) {
+            $transport->setToken($token);
+        }
+
```

Comment:
Should we verify that username, password AND token is not provided at the same time?

Reply:
That seems like an option and would probably also be an answer to @smnandre 's point in https://github.com/symfony/symfony/pull/53594#issuecomment-1902263539 ("With your implementation, we store two exclusive ways of authorization, that cannot be used together.")